pair_data_aggregation_service.py

```diff
@@ -490,25 +493,71 @@ async def process_pairs_trade_volume_and_reserves(
             )
             if not dag_chain_7d:
                 return
-
-            # calculate trade volume 24h
-            pair_trade_volume_24h = calculate_pair_trade_volume(dag_chain_24h)
-
-            # calculate trade volume 7d
-            pair_trade_volume_7d = calculate_pair_trade_volume(dag_chain_7d)
             dag_chain_24h_intact_payloads = [x for x in dag_chain_24h if x is not None and x.get('data') and x['data'].get('payload')]
+            dag_chain_24h_bad_payloads = [x for x in dag_chain_24h if x is None or not x.get('data') or not x['data'].get('payload')]
             dag_chain_7d_intact_payloads = [x for x in dag_chain_7d if x is not None and x.get('data') and x['data'].get('payload')]
+            dag_chain_7d_bad_payloads = [x for x in dag_chain_7d if x is None or not x.get('data') or not x['data'].get('payload')]
+            if dag_chain_24h_bad_payloads:
+                logger.warning(
+                    "Found %s bad payloads in 24h dag-chain | projectId: %s",
+                    dag_chain_24h_bad_payloads,
+                    project_id_trade_volume
+                )
+                slack_alert_msg = {
```

Comment:
There is a lot of code repetition for sending alerts in `pair_data_aggregation_service` and `proto_sliding_window_cacher_service`. It might be a good idea to refactor and move the alert sending code to a function in utils. Something like this should do the trick
```python
def send_slack_alerts(severity:str, service_name:str, errorDetails: str, logger_error_if_failed:str):
    ...
```

Reply:
@anomit will raise a separate PR to clean it up. Approving for now!